src/components/atoms/chef/CircleChef.tsx

```diff
@@ -0,0 +1,17 @@
+import Image from "next/image";
+
+interface CircleChefProps {
+  icon: string;
+  name: string;
+}
+
+export const CircleChef = (props: CircleChefProps) => {
+  return (
+    <div className="flex flex-col items-center">
```

Comment:
```suggestion
    <Link href={props.url} className="flex flex-col items-center">
```
Linkコンポーネントにしておくとよさそうです！

Reply:
対応いたしました！